doc-Managing_Providers/topics/Enabling_Red_Hat_Virtualization_CU.adoc

```diff
@@ -4,7 +4,7 @@
 Configure the following to collect capacity and utilization data from a Red Hat Virtualization provider:
 
 * In {product-title_short}, enable the capacity and utilization server roles from the settings menu, in menu:Configuration[Server > Server Control]. For more information on capacity and utilization collection, see https://access.redhat.com/documentation/en-us/red_hat_cloudforms/4.5/html-single/deployment_planning_guide/#assigning_the_capacity_and_utilization_server_roles[Assigning the Capacity and Utilization Server Roles] in the _Deployment Planning Guide_.
-
+* For information on selecting clusters and datastores used to collect data, see https://access.redhat.com/documentation/en-us/red_hat_cloudforms/4.5/html-single/deployment_planning_guide/#enabling-a-cluster,-host,-or-datastore-for-capacity-and-utilization-collection[Enabling a Cluster, Host, or Datastore for Capacity and Utilization Collection] in the _General Configuration Guide_.
```

Comment:
This link goes to the wrong guide/section. Also, we should write/markup "in the _General Configuration Guide_." as "in the _General Configuration_ guide." to be consistent.